extension/src/Common.ts

```diff
@@ -67,3 +67,16 @@ export function mkDirByPathSync(targetDir: string) {
 }
 
 
+export function formatToday() {
+  var d = new Date(),
+    month = '' + (d.getMonth() + 1),
+    day = '' + d.getDate(),
+    year = d.getFullYear();
+
+  if (month.length < 2)
+    month = '0' + month;
+  if (day.length < 2)
+    day = '0' + day;
```

Comment:
no brackets? I don't think we have a rule against this but it's not a common style in this repo. Mostly I'm surprised that your es-/tslint approves this :D

Reply:
copy&paste from SO, was so happy that it worked that I forgot to clean it up. But as said, since I was not finnished yet - I haven't yet reviewed the PR code myself. (no guarantee that I would have changed this, though! 😝)